### PR TITLE
feat(workflows): deliver issue-to-PR workflows and scaling roadmap

### PR DESCRIPTION
**Pull request by owainlewis**
Add two local coding workflows and the roadmap for scaling them through Machinist. The stdin flow creates an isolated worktree, runs implementation and independent review, then opens a PR. The issue launcher accepts a GitHub issue URL, validates the returned open PR and its issue linkage, waits for CI/review feedback, and allows at most three feedback passes.

The launcher uses the installed Codex CLI with a pinned Python SDK, streams progress to stderr, and returns one JSON result on stdout. It filters known review-status notices, waits for active Codex reviews of the current head, assesses only new comments, and avoids raw failed-command output in logs. Newly failing checks can enter a repair pass even when the preceding assessment needed no commit. The stdin flow strips credentials from the remote address passed to GitHub CLI.

The integration also includes the cron unit-step fix delivered by the live test run: explicit single-value `/1` schedules expand through the field maximum, while plain values retain their behavior.

Validation:

- Full `just check` passed on integration commit a8d6dfe: 116 Python evals, 7 flow integration tests, 36 frontend tests, frontend build, Go formatting, vet, race tests, and build. The final commit changes only one roadmap sentence.
- Ruff and `git diff --check` pass.
- Live read-only validation of PR #475 passed, including mixed-case issue URL handling and current-head feedback collection.
- Fresh independent review approved the code fixes and the final integration at 7baba43.
- Earlier live runs produced PRs #466 and #475. The recorded run did not exercise a genuine code-repair cycle; no-op prompt compliance remains to be tested in a subsequent live run.

This preserves and integrates the commits from #465, #467, #468, and #475. #466 is already on main. The combined PR gives the completed work one validation against the latest main before merging.

The workflows remain local prototypes: agents have the configured full host access, some build/review evidence is agent-reported, and visible-check polling does not guarantee discovery of future workflows or human reviews. The roadmap describes the remaining execution, recovery, and scaling work.

### CODE REVIEW

File: agent.py [hunk 1]
```diff
@@ -126,18 +127,72 @@ def log_feedback(feedback: dict) -> None:
...
+            log(f"Running: {item.command}")
+        else:
+            outcome = (
+                f"exit {item.exit_code}"
+                if item.exit_code is not None
+                else item.status.value
+            )
+            log(f"Command finished: {outcome}")
+            if item.exit_code != 0 and item.aggregated_output:
+                log(f"Command output:\n{item.aggregated_output}")
```
[the remark this answers is not in the document]
Comment (owainlewis) [reply agent.py]: Fixed in https://github.com/owainlewis/machinist/commit/22bac09. Failed-command progress now reports the exit status and leaves raw command output with the agent for diagnosis. The streaming regression asserts that a fake token in failed output never reaches stderr. All 57 agent tests and Ruff pass; independent review approved. This removes the direct raw-output dump; it does not claim general secret detection for arbitrary agent prose.
